container.ks.in

```diff
@@ -31,6 +31,7 @@ clearpart --all
 -kernel
 -systemd
 bash
+rpm
```

Comment:
I think that this needs to be fixed in Anaconda. @VladimirSlavik, could you look at it, please? We should probably check that `rpm` is available on the target system before we try to use it to import GPG keys.

Reply:
Ha, thank you! Fixing...